openstack/cinder/templates/backup-deployment.yaml

```diff
@@ -0,0 +1,101 @@
+kind: Deployment
+apiVersion: extensions/v1beta1
+
+metadata:
+  name: cinder-backup
+  labels:
+    system: openstack
+    type: backend
+    component: cinder
+spec:
+  replicas: {{ .Values.pod.replicas.backup }}
+  revisionHistoryLimit: {{ .Values.pod.lifecycle.upgrades.deployments.revisionHistory }}
+  strategy:
+    type: {{ .Values.pod.lifecycle.upgrades.deployments.podReplacementStrategy }}
+    {{ if eq .Values.pod.lifecycle.upgrades.deployments.podReplacementStrategy "RollingUpdate" }}
+    rollingUpdate:
+      maxUnavailable: {{ .Values.pod.lifecycle.upgrades.deployments.rollingUpdate.maxUnavailable }}
+      maxSurge: {{ .Values.pod.lifecycle.upgrades.deployments.rollingUpdate.maxSurge }}
+    {{ end }}
+  selector:
+    matchLabels:
+      name: cinder-backup
+  template:
+    metadata:
+      labels:
+        name: cinder-backup
+{{ tuple . "cinder" "backup" | include "helm-toolkit.snippets.kubernetes_metadata_labels" | indent 8 }}
+      annotations:
+        configmap-etc-hash: {{ include (print $.Template.BasePath "/etc-configmap.yaml") . | sha256sum }}
+{{- if and (eq .Capabilities.KubeVersion.Major "1") (lt .Capabilities.KubeVersion.Minor "5") }}
+        scheduler.alpha.kubernetes.io/affinity: >
+            {{ tuple . "cinder" "backup" | include "kubernetes_pod_anti_affinity" }}
+{{- end }}
+    spec:
+{{- if and (eq .Capabilities.KubeVersion.Major "1") (ge .Capabilities.KubeVersion.Minor "7") }}
+{{ tuple . "cinder" "backup" | include "kubernetes_pod_anti_affinity" | indent 6 }}
+{{- end }}
+      containers:
+        - name: cinder-backup
+          image: {{.Values.global.imageRegistry}}/{{.Values.global.image_namespace}}/ubuntu-source-cinder-backup:{{.Values.imageVersionCinderVolumeBackup | default .Values.imageVersion | required "Please set cinder.imageVersion or similar" }}
+          imagePullPolicy: IfNotPresent
+          command:
+            - kubernetes-entrypoint
+          env:
+            - name: COMMAND
+              value: "cinder-backup"
+            - name: NAMESPACE
+              value: {{ .Release.Namespace }}
+            - name: DEPENDENCY_SERVICE
+              value: "cinder-postgresql,rabbitmq"
+            - name: SENTRY_DSN
+              value: {{.Values.sentry_dsn | quote}}
+            - name: PGAPPNAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+          livenessProbe:
+            httpGet:
+              path: /
+              port: cinder-backup
+            initialDelaySeconds: 15
+            timeoutSeconds: 5
+          readinessProbe:
+            httpGet:
+              path: /healthcheck
+              port: cinder-backup
+            initialDelaySeconds: 15
+            timeoutSeconds: 5
+          volumeMounts:
+            - name: cinder-etc
+              mountPath: /tmp/cinder-backup.sh
```

Comment:
Can you remove the script (also from the chart and the configmap)? It is not needed